40/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:stretch-slim
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates git wget unzip xz-utils pylint \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG SONAR_SCANNER_HOME=/opt/sonar-scanner
+ARG NODEJS_HOME=/opt/nodejs
+
+ENV SONAR_SCANNER_HOME=${SONAR_SCANNER_HOME} \
+    SONAR_SCANNER_VERSION=4.0.0.1744 \
+    NODEJS_HOME=${NODEJS_HOME} \
+    NODEJS_VERSION=v8.12.0 \
+    PATH=${PATH}:${SONAR_SCANNER_HOME}/bin:${NODEJS_HOME}/bin \
+    NODE_PATH=${NODEJS_HOME}/lib/node_modules
+
+WORKDIR /opt
+RUN wget -U "scannercli" -q -O /opt/sonar-scanner-cli.zip https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${SONAR_SCANNER_VERSION}-linux.zip \
+    && unzip sonar-scanner-cli.zip \
+    && rm sonar-scanner-cli.zip \
+    && mv sonar-scanner-${SONAR_SCANNER_VERSION}-linux ${SONAR_SCANNER_HOME} \
+    && wget -U "nodejs" -q -O nodejs.tar.xz https://nodejs.org/dist/${NODEJS_VERSION}/node-${NODEJS_VERSION}-linux-x64.tar.xz \
+    && tar Jxf nodejs.tar.xz \
+    && rm nodejs.tar.xz \
+    && mv node-${NODEJS_VERSION}-linux-x64 ${NODEJS_HOME} \
+    && npm install -g typescript
+
+WORKDIR /usr/src
```

Comment:
since this command is irrelevant to the following ones and only relevant to the execution, why not put it at the end of the file, after or before the `ENTRYPOINT`?

Reply:
Changed 👍